docs/liquidation-helper.md

```diff
@@ -0,0 +1,57 @@
+
+## Liquidation Methods
+
+A position can be liquidated using two methods:
+
+1. **Max Liquidation**: The easiest method is to liquidate the entire position.
+2. **Arbitrary Borrow Part Liquidation**: The other method requires providing a borrow part, which can be obtained by calling the `userBorrowPart` function on the specific cauldron to liquidate.
+
+In both methods, MIM must be approved to be spent on `0x82e07721837a740985186663b66423b0741960b6`.
+
+## Approving MIM
+
+To approve MIM on `0x82e07721837a740985186663b66423b0741960b6`, go to Etherscan or the equivalent for other chains and go to the MIM contract address.
+
+MIM addresses are the following:
+
+- Mainnet: `0x99D8a9C45b2ecA8864373A26D1459e3Dff1e17F3`
+- Optimism: `0xB153FB3d196A8eB25522705560ac152eeEc57901`
+- Avalanche: `0x130966628846BFd36ff31a822705796e8cb8C18D`
+- Arbitrum: `0xFEa7a6a0B346362BF88A9e4A88416B77a57D6c2A`
+- BSC: `0xfE19F0B51438fd612f6FD59C1dbB3eA319f433Ba`
+
+In the `Contract > Write Contract` tab, locate the `approve` function and enter the following:
+
+- Spender: `0x82e07721837a740985186663b66423b0741960b6`
+- Value: `115792089237316195423570985008687907853269984665640564039457584007913129639935`
+
+Note that the `115792089237316195423570985008687907853269984665640564039457584007913129639935` is the maximum value, so there's no limit. Any reasonable amount can be used there. For example, `1000000000000000000000` is `1000k`. The important thing is that the number must be 18 decimals.
+
+Then click the `Write` button to send the transaction. After the transaction is completed, MIM is approved to be used to liquidate Abracadabra positions.
+
+## Previewing Liquidations
+
+On `0x82e07721837a740985186663b66423b0741960b6` on `Contract > Read Contract`, there's `preview` functions that indicate, for a given cauldron address, account, and optional borrow part in case of arbitrary borrow part liquidation, if a position is liquidable. It also shows the expectable required MIM to liquidate and the expected collateral amount that will be returned if the liquidation succeed. This can be used to validate and approximate the outcome of the liquidations.
+
+> Note that Tenderly can also be used to simulate transactions and get more details about them.
+
+## Liquidation Functions
+
+Once a liquidable position has been located, the following functions can be used to liquidate the position:
+
+- `liquidate`: liquidate an account position on the given cauldron, for the given borrow part. The cauldron version can be found by going to the cauldron etherscan (or equivalent) and looking at the contract name. For example, if the name is `CauldronV2Flat` the cauldron version is 2, `CauldronV4`, 4 and so on. As of now, most cauldrons use version 2, 3 and 4.
+
+- `liquidateMax`: same as `liquidate` function except borrow part doesn't need to be mentioned as it's going to take the full liquidate amount.
```

Comment:
```suggestion
- `liquidateMax`: same as `liquidate` function except the borrow part is not required as it will attempt to liquidate the full loan amount.
```